lib/oregon_digital/derivatives/processors/processor.rb

```diff
@@ -0,0 +1,15 @@
+module OregonDigital::Derivatives::Processors
+  class Processor
+    def temporary_file
+      tempfile = Tempfile.new(["pyramidal_tmp","#{File.extname(file.path)}"])
```

Comment:
Is this pyramidal-only code?  If so, maybe the class name should be changed; if not, this line is a little misleading.

Also, I'm not sure the value of "#{...}" - am I missing something in that second array element?